nucypher/utilities/networking.py

```diff
@@ -147,9 +152,9 @@ def get_external_ip_from_known_nodes(known_nodes: FleetSensor,
     if len(known_nodes) < sample_size:
         return  # There are too few known nodes
     sample = random.sample(list(known_nodes), sample_size)
-    client = NucypherMiddlewareClient()
+    client = NucypherMiddlewareClient(eth_provider_uri=provider_uri)
```

Comment:
how come this one is still `eth_provider_uri`?

Reply:
One of the many naming discussions of the times :-)  This parameter will always be a reference to the chain that Ursula's are staking on, which I assert will be ethereum for the foreseeable future.